spec/models/user_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before(:each) do
+    @user = User.create(name: 'Kevin', photo: 'www.photo.com',
+                        bio: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aut officia repudiandae exe.',
+                        posts_counter: 2)
+  end
+
+  it 'name should be present' do
+    @user.name = nil
+    expect(@user).to_not be_valid
+  end
+
+  it 'posts_counter should be greater than zero' do
+    @user.posts_counter = 0
+    expect(@user).to_not be_valid
```

Comment:
* Kindly fix this test as it is failing when i run `rspec spec/models/*.rb`
![image](https://user-images.githubusercontent.com/52497246/140444940-dd1be00e-20d1-4917-a638-d931c2c4ef65.png)

Reply:
Thanks for the review, @Davidosky007. I already made the changes.